mesheryctl/internal/cli/root/system/restart.go

```diff
@@ -102,32 +98,12 @@ func restart() error {
 				return ErrRestartMeshery(err)
 			}
 		} else {
-			// create a kubernetes client
-			client, err := meshkitkube.New([]byte(""))
-
-			if err != nil {
-				return err
-			}
-
-			// Create a pod interface for the MesheryNamespace
-			podInterface := client.KubeClient.CoreV1().Pods(utils.MesheryNamespace)
-
-			// List the pods in the MesheryNamespace
-			podList, err := podInterface.List(context.TODO(), v1.ListOptions{})
-			if err != nil {
-				return err
+			if err := stop(); err != nil {
+				return ErrRestartMeshery(err)
```

Comment:
I think we can remove the whole switch-case logic here, as we are doing same actions in both the cases.

Reply:
@piyushsingariya  @navendu-pottekkat  I think we should enable skipUpdateFlag when starting?

Reply:
`start()` function should handle the `skipUpdateFlag` right?

We should definitely enable it and we should pass the responsibility of acknowledging the flag to `start()`if we haven't already.